lib/screens/feed.dart

```diff
@@ -79,4 +108,11 @@ class Feed extends StatelessWidget {
       ),
     ));
   }
+
+  Future<void> _determinePosition() async {
+    var position = await Geolocator.getCurrentPosition();
```

Comment:
Can you replace this with the center location of the map? I think that's more natural that I can see the photos sorted by map location instead of my current location